hack/components/components.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source hack/components/common_functions.sh
+CNAO_VERSION=${VERSION} # Exported from Makefile
+
+echo 'Setup temporary Go path'
+export GOPATH=${PWD}/_components/go
+mkdir -p $GOPATH
+export PATH=${GOPATH}/bin:${PATH}
+
+echo 'kubemacpool'
```

Comment:
maybe worth to create array of the components and a function that will echo and then source them ?
just those 3 we have atm ?